File: Lesson2/Car.py
from Lesson2.Vehicle import Vehicle


class Car(Vehicle):
    def __init__(self):
        self.num_wheels = 4
        self.speed = 0

    def get_num_wheels(self):
        return self.num_wheels

    def get_speed(self):
        return self.speed

    def test_drive(self):
        self.speed = 60

    def park(self):
        self.speed = 0

File: Lesson2/hw2/src/second/hw/Car.java
package second.hw;

public class Car extends Vehicle {
    private int numWheels;
    private int speed;

    public Car() {
        this.numWheels = 4;
        this.speed = 0;
    }

    public int getNumWheels() {
        return numWheels;
    }

    public int getSpeed() {
        return speed;
    }

    public void testDrive() {
        this.speed = 60;
    }

    public void park() {
        this.speed = 0;
    }
}

